app/controllers/coronavirus_form/offer_space_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::OfferSpaceController < ApplicationController
+  include ActionView::Helpers::SanitizeHelper
+  include FieldValidationHelper
+
+  def show
+    render "coronavirus_form/#{PAGE}"
+  end
+
+  def submit
+    offer_space = sanitize(params[:offer_space]).presence
+    session[:offer_space] = offer_space
+
+    invalid_fields = validate_radio_field(
+      PAGE,
+      radio: offer_space,
+    )
+
+    if invalid_fields.any?
+      flash.now[:validation] = invalid_fields
+      render "coronavirus_form/#{PAGE}"
+    elsif session["check_answers_seen"]
+      redirect_to controller: "coronavirus_form/check_answers", action: "show"
+    else
+      redirect_to controller: "coronavirus_form/offer_advice", action: "show"
```

Comment:
This will need to be changed to the 'what kind of space' controller

Reply:
Ah, you chose "offer advice". I went with "expert advice"?

Reply:
@issyl0 feel free to change this in your PR.  Mine was just a placeholder so I could add a test for the redirect.